app/parse.py

```diff
@@ -14,5 +34,72 @@ class Course:
     type: CourseType
 
 
+COURSE_FIELDS = [field.name for field in fields(Course)]
+
+
+def parse_single_course(course_soup, course_type: CourseType) -> Course:
+    return Course(
+        name=course_soup.select_one(".mb-16")
+        .text.replace("Курс ", "")
+        .replace(" Вечірній", ""),
+        short_description=course_soup.select_one(
+            ".CourseCard_courseDescription__Unsqj"
+        ).text,
+        type=course_type,
+    )
+
+
+def get_full_time_courses(full_time_soup: BeautifulSoup) -> [Course]:
+    courses = full_time_soup.select(".CourseCard_cardContainer__7_4lK")
+    return [
+        parse_single_course(course, CourseType.FULL_TIME)
+        for course in courses
+    ]
+
+
+def get_part_time_courses(part_time_soup: BeautifulSoup) -> [Course]:
+    courses = part_time_soup.select(".CourseCard_cardContainer__7_4lK")
+    return [
+        parse_single_course(course, CourseType.PART_TIME)
+        for course in courses
+    ]
```

Comment:
Don't repeat yourself. Find a way to make one function instead of two almost identical